README.md

```diff
@@ -298,14 +299,17 @@ All the data (past and coming) are available publicly in JSON:
 * 2: [Stir Trek 2025](https://stirtrek.com) - Columbus, OH (USA)
 * 2: [BSides Groningen](https://bsidesgrunn.org) - Groningen (Netherlands) <a href="https://sessionize.com/bsidesgrunn/"><img alt="CFP BSides Groningen" src="https://img.shields.io/static/v1?label=CFP&message=until%2028-February-2025&color=green"></a>
 * 3: [KCD Costa Rica](https://community.cncf.io/events/details/cncf-kcd-costa-rica-presents-kcd-costa-rica-2025/) - Heredia (Costa Rica) <a href="https://sessionize.com/kcd-costa-rica-2025/"><img alt="CFP KCD Costa Rica" src="https://img.shields.io/static/v1?label=CFP&message=until%2009-February-2025&color=green"></a>
+* 4-5: [KCD New York](https://kcdnewyork.com) - New York, NY (USA) <a href="https://sessionize.com/kcd-new-york-2025"><img alt="CFP KCD New York 2025" src="https://img.shields.io/static/v1?label=CFP&message=until%2028-February-2025&color=green"></a>
```

Comment:
Already exists but with a wrong date, can you update the existing one with the good date please? :)